src/main.py

```diff
@@ -27,16 +30,56 @@
 CORS(app)
 setup_admin(app)
 app.cli.add_command(init_db)
+
 @app.route('/register', methods=['POST'])
 def create_user():
     body=request.get_json()
     try:
-        new_user= User(email=body["email"], password=body["password"], is_active=body["is_active"], name=body["name"], last_name=body["last_name"], phone=body["phone"], location=body["location"], biografy=body["biografy"])
+        hashed_password = generate_password_hash(body['password'], method='sha256')
+        print(hashed_password,"estoy en hashedpassword register")
+        new_user= User( email=body["email"], password=hashed_password, is_active=True, name=body["name"], last_name=body["last_name"], phone=body["phone"], location=body["location"], biografy=body["biografy"])
+        #, phone=body["phone"], location=body["location"], biografy=body["biografy"]
+        print(new_user,"soy new user")
         new_user.create_user()
         return jsonify(new_user.serialize()), 200
     except:
         return "Couldn't create the user",409
 
+@app.route('/login', methods=['GET','POST'])  
+def login_user(): 
+ 
+    auth = request.authorization   
+
+    if not auth or not auth.username or not auth.password:  
+        return 'could not verify', 401, {'WWW.Authentication': 'Basic realm: "login required"'}    
+
+    user = User.query.filter_by(email=auth.username).first()   
+        
+    if check_password_hash(user.password, auth.password):  
+        token = jwt.encode({'id': user.id, 'exp' : datetime.datetime.utcnow() + datetime.timedelta(minutes=30)}, app.config['SECRET_KEY'])  
+        return jsonify({'token' : token.decode('UTF-8')}) 
+
+    return make_response('could not verify',  401, {'WWW.Authentication': 'Basic realm: "login required"'})
+
+@app.route('/users', methods=['GET'])
```

Comment:
La lógica del get all users debería estar en el modelo, pero creo que no necesitáis este endpoint